portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,60 @@
+
+  
+#fact-container {
+    font-size: 120%;
```

Comment:
please use 2 spaces of indentation here and below:
https://google.github.io/styleguide/htmlcssguide.html#Indentation